test/src/runner/testReporter.js

```diff
@@ -364,4 +364,32 @@ describe('testReporter', function() {
     }
     assert.strictEqual(possibleError, null);
   });
+
+  it('test with multiple reporters', function() {
+    mockery.registerMock('nightwatch_reporter', {
+      async write(_results, _options) {
+
+        return 'nightwatch_reporter_output';
+      }
+    });
+    mockery.registerMock('html_reporter', {
+      async write(_results, _options) {
+
+        return 'html_reporter_output';
+      }
+    });
+
+    const reporter = new Reporter('nightwatch_reporter,html_reporter', {
+      globals: {
+        reporter(_results, done) {
+          done();
+        }
+      },
+      output_folder: 'output'
+    });
+
+    return reporter.writeReportToFile().then(function(result) {
```

Comment:
Ideally we should test if the file gets written. Try mocking the `fs` functions rather than the entire write function.

Reply:
can be added later